src/Uno.UI/BaseActivity.Android.cs

```diff
@@ -67,7 +67,8 @@ public static class TraceProvider
 		/// <summary>
 		/// Gets a list of all activities which are currently alive.
 		/// </summary>
-		public static IImmutableDictionary<int, BaseActivity> Instances => _instances;
+		public static IImmutableDictionary<int, BaseActivity> Instances
+			=> ImmutableDictionary<int, BaseActivity>.Empty.AddRange(_instances) ;
```

Comment:
```suggestion
			=> _instances.ToImmutableDictionary() ;
```

(But I'm pretty sure it will conflict with the uno one ^^)